internal/blobstore/multipart.go

```diff
@@ -131,16 +132,16 @@ func (s *Store) NewUpload(expires time.Time) (uploadId string, err error) {
 // PutPart returns an error with an ErrNotFound cause.
```

Comment:
// If one of the parameters is badly formed, it returns an error with an ErrBadParams cause.